### DISCUSSION TITLE
Can I hide display name of field or part,please

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#15021] [Q&A] [answered]
Can I hide display name of field or part ,please?

I hope I can hide display name of field or part by two method:

1 During to add field or part in content type definition, it maybe to add a configuration item "allow hide display name";
2 During to custom content template, it maybe to add function parameter "allow hide display name" in @DisplayAsync(), like following:
 @await DisplayAsync(Model.Content["ContentType-FieldName"], AllowHideDisplayName=true)

Thanks.

**Comment by MikeAlhayek** [comment]
Where do you want to hide the display name from? If you want to hide the display name in the editor template, how will the user know what does the field represent? If you want to hide the display name from showing up in a display template, you may do that in the display template

**Reply by effapp** [reply]
Hi,
I wanna hide the display name of MediaField from detail display type only, but don't hide in the editor template.

**Reply by MikeAlhayek** [reply, marked as answer]
@effapp create a template in your custom theme that looks like this

You can name it the same to override all media fields. You'll just remove the content on line 12

https://github.com/OrchardCMS/OrchardCore/blob/16ade0f2e387bc3d6c83f7d66b8e3489ca9e057b/src/OrchardCore.Modules/OrchardCore.Media/Views/MediaField.cshtml#L4

**Comment by emrahtokalak** [comment]
You can change the appearance of a content type using placement in the admin panel. If there is an area you want to hide, you can do it using the same method.

https://docs.orchardcore.net/en/main/docs/reference/core/Placement/

**Comment by MikeAlhayek** [comment]
@emrahtokalak not in this case. He want to hide partial of a content. There is an item in my list "at some point I'll get to it" which allows us to control how to render the display name and the format of the value
